packages/paste-core/components/chat-log/__tests__/customization.spec.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import {screen, render} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {ChatMessage, ChatBubble} from '../src';
+
+describe('Customization', () => {
```

Comment:
suggestion: we do tend to have some very explicit tests around the value of the data attribute, essentially ensuring that name / identifier doesn't changes as it's a public API.